docs/man_pages/project/testing/debug-android.md

```diff
@@ -0,0 +1,35 @@
+debug android
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder debug android`
+
+On Windows opens up AppBuilder Windows Debugger program. On Mac the Chrome Inspector address is copied to your clipboard and you just need to open Chrome and paste it in the address bar.
```

Comment:
We can use our context specific help here, for example I imagine something like this:
```JavaScript
<% if(isConsole) { %>
<% if(isWindows) { %> Shows the debug tools to let you debug applications on connected Android devices. <% } %>
<% if(isMacOS) { %> Allows you to debug your Cordova applications in Google Chrome, by opening device port and copying the required link to your clipboard. You have to paste it manually in your browser. <% } %>
<% } %>
<% if(isHtml) { %>
On Windows shows the debug tools to let you debug applications on connected Android devices. On OS X you can debug your Cordova applications in Google Chrome. The command will open device port and copy the required link to your clipboard. You have to paste it manually in your browser.
<% } %>
```